packages/design/src/icons/apple.svg

```diff
@@ -1,3 +1,3 @@
-<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
For some reason, the previous Apple icon was 150x150. Do you think we should keep the same size for the same version?